dashboard/src/app/admin/user-management/user-details/user-details.html

```diff
@@ -0,0 +1,45 @@
+<che-toolbar che-title="User details"
```

Comment:
Better to show user name instead of "User details" to be consistent with other pages like that.